includes/functions-bindings-event-hubs.md

```diff
@@ -146,6 +147,7 @@ The [host.json](../articles/azure-functions/functions-host-json.md#eventhub) fil
 |Property  |Default | Description |
 |---------|---------|---------|
 | maxEventBatchSize| 10| The maximum number of events that will be included in a batch for a single invocation. Must be at least 1.|
+| targetUnprocessedEventThreshold | | The desired number of unprocessed events per Function instance.  This is used in target-based scaling to override the default scaling threshold inferred from the `maxEventBatchSize` option. If set, the total unprocessed event count will be divided by this value to determine the number of Function instances needed, which is rounded up to a count that creates a balanced partition distribution.|
```

Comment:
```suggestion
| targetUnprocessedEventThreshold | n/a | The desired number of unprocessed events per Function instance.  This is used in target-based scaling to override the default scaling threshold inferred from the `maxEventBatchSize` option. When set, the total unprocessed event count is divided by this value to determine the number of function instances needed, which is rounded up to a count that creates a balanced partition distribution.|
```
Let's make the default `n/a` so that we're clear there's not one.

Reply:
I'd prefer not to.  This would make the entry for `targetUnprocessedEventThreshold` inconsistent with the others.  In addition, if I recall, we explicitly exchanged the `n/a` values with an empty cell due to customer confusion.

I'd be open to something like `null` or  `<None>` similar, if applied consistently.   I'm not a fan of `n/a`, however, as "not applicable" doesn't fit the context for a parameter value.

Reply:
Your choice. I just don't like the empty cell. 😉

Reply:
Updated to `null` for values with null defaults; suggestions for formatting and phrasing applied.